src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -121,8 +126,15 @@ public void loop() {
       mTurretLogger.verbose("movement blocked");
       OperatorInterface.getInstance().setOperatorRumble(true);
     }
-
     // run the PIDSubsystem system's loop
     this.periodic();
   }
+
+  public void setCameraLight(boolean on) {
+    if (on) {
```

Comment:
Change to a ternary operator:
`cameraLight.set(on ? Relay.Value.kOn : Relay.Value.kOff);`

Reply:
@onedeadeye > done